exchange/bitswap/workers.go

```diff
@@ -182,10 +182,11 @@ func (bs *Bitswap) rebroadcastWorker(parent context.Context) {
 	defer cancel()
 
 	broadcastSignal := time.After(rebroadcastDelay.Get())
+	tick := time.Tick(10 * time.Second)
```

Comment:
Eh, not really needed. This is a single goroutine that lasts the entire lifetime of the executable.